test/db/cmd/dwarf

```diff
@@ -1782,7 +1782,7 @@ EXPECT=<<EOF
 |           ; arg SmallInt mag @ stack - 0x10
 |           ; arg SmallInt szel @ stack - 0x18
 |           ; var SmallInt i @ stack - 0x1c
-|           0x00401980      55             push  rbp                   ; void palya(SmallInt mag, SmallInt szel)
+|           0x00401980      55             push  rbp                   ; unit p$mozgkigyo palya(smallint,smallint)
```

Comment:
i think the original one is better.

Reply:
It's duplicates the signature few lines before

Reply:
What is this comment in after, is it a function name without demangle?

Reply:
kinda weird still that this push has this comment

Reply:
@imbillow yes, it's the mangled function name.

Reply:
@ret2libc @wargio please check the original issue. The information was *duplicated*, but now it's less so. The comment was completely redundant since it's already in the function name/signature.